### ISSUE TITLE
Horizontal label scrolls outside of user view region (Google Chrome)

### ISSUE THREAD

**Issue by axdemelas** [ahlechandre/mdl-stepper#11]
Horizontal steps uses `overflow: hidden` to hide the scrollbars. Now step label scrolling outsite of users view on Google Chrome when text box inside content changes. 

I really don't know if it's a plataform feature or bug.

Chromium issue: [https://bugs.chromium.org/p/chromium/issues/detail?id=116655](https://bugs.chromium.org/p/chromium/issues/detail?id=116655)

Screenshots of the behavior in major browsers above.
#### Google chrome before change input text.

![overflow-hidden-scrolling-chrome](https://cloud.githubusercontent.com/assets/7119058/15993393/ff20ea86-30ba-11e6-9a15-a088d0460e8e.png)
#### Google chrome after change input text.

![overflow-hidden-scrolling-chrome-bug](https://cloud.githubusercontent.com/assets/7119058/15993390/fe6a9376-30ba-11e6-9255-cf2cf47a8d71.png)
#### Microsoft Edge after change input text.

![overflow-hidden-scrolling-edge](https://cloud.githubusercontent.com/assets/7119058/15993391/fea2ecbc-30ba-11e6-8483-eb2dc9fd95ea.png)
#### Firefox after change input text.

![overflow-hidden-scrolling-firefox](https://cloud.githubusercontent.com/assets/7119058/15993392/fee2f046-30ba-11e6-8759-767dfe97c85e.png)
#### Browsers info
- **Google Chrome**: 52.0.2743.33 beta-m (64-bit)
- **Microsoft Edge**: 25.10586.0.0
- **Firefox**: 46.0.1

**Comment by axdemelas** [comment]
The polyfill to repair the behavior could be:

``` js
// Selects the stepper.
var stepperElement = document.querySelector('.mdl-stepper');
// Get all steps.
var steps = stepperElement.querySelectorAll('.mdl-step');

for (var i = 0; i < steps.length; i++) {

   steps[i].addEventListener('scroll', function (event) {
       // Forcing the current step to keep its scroll position.
       event.target.scrollTo(0, 0);
   });
}
```

I might include this on next javascript production version (`v1.1.1`).

**Comment by axdemelas** [comment]
Properly fix on #17
